### PR TITLE
Adding gdecomp

### PR DESCRIPTION
**Pull request by floriandeboissieu**
My first PR to add a package to conda-forge.
Any recommandation is welcomed.

Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [x ] Build number is 0
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by floriandeboissieu** [comment]
@conda-forge/help-python-c new gdecomp package is ready for review

File: recipes/gdecomp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+  run:
+    - cython
+    - numpy
```
Comment (marcelotrevisani) [inline recipes/gdecomp/meta.yaml:26]:
```suggestion
    - {{ pin_compatible('numpy') }}
```

File: recipes/gdecomp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/gdecomp/meta.yaml:23]:
```suggestion
    - python
    - setuptools >=18.0
```

File: recipes/gdecomp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 11b88af3cc276376ed45f48778fbc8399e2789720c3b83e3c4e8af74217418c0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/gdecomp/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/gdecomp/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    - numpy
+    - python
+
+test:
+  imports:
+    - gdecomp
+
+about:
+  home: https://gitlab.com/pytools4dart/gdecomp
+  license: GNU General Public v3 or later (GPLv3+)
```
Comment (marcelotrevisani) [inline recipes/gdecomp/meta.yaml:35]:
```suggestion
  license: GPL-3.0-or-later
```

**Comment by floriandeboissieu** [comment]
@marcelotrevisani thanks for the proposals, all accepted

**Pull request merged**